terraform/s3-export-download-bucket.tf

```diff
@@ -6,6 +6,9 @@ resource "cloudfoundry_service_instance" "beis-roda-s3-export-download-bucket" {
   name         = "beis-roda-${var.environment}-s3-export-download-bucket"
   space        = cloudfoundry_space.space.id
   service_plan = data.cloudfoundry_service.aws-s3-bucket.service_plans["default"]
+  json_params  = <<EOF
+  {"public_bucket":true}
+EOF
```

Comment:
Is this meant to be in here? Was this superseded by Olivia and Ash's work?

Reply:
The original bucket is public, so I guess it's ok?